packages/schema/bind/src/bindings/wasm-as/index.ts

```diff
@@ -6,23 +6,27 @@ import {
   parseSchema,
   extendType,
   addFirstLast,
-  toGraphQLType,
+  toPrefixedGraphQLType,
 } from "@web3api/schema-parse";
 import path from "path";
 import Mustache from "mustache";
 
 export function generateBinding(schema: string): OutputDirectory {
   const entries: OutputEntry[] = [];
   const typeInfo = parseSchema(schema, {
-    transforms: [extendType(Functions), addFirstLast, toGraphQLType],
+    transforms: [extendType(Functions), addFirstLast, toPrefixedGraphQLType],
   });
 
+  const absolutePath = path.join(__dirname, "./templates");
+  const directory = readDirectory(absolutePath);
+  const subTemplates = loadSubTemplates(directory.entries);
```

Comment:
Thank you so much for doing this @mpetrun5!!! This removed a lot of duplicate code :sweat_smile: